### ISSUE TITLE
DHF 5.0.0 - Indexes defined in final-database.json not being deployed during mlDeploy

### ISSUE THREAD

**Issue by derms** [marklogic/marklogic-data-hub#2529]
#### The issue

Indexes defined in src/main/ml-config/databases/final-database.json not being deployed during mlDeploy

Sample configuration below -

```
{
   "database-name":"%%mlFinalDbName%%",
   "range-element-index":[

   ],
   "schema-database":"%%mlFinalSchemasDbName%%",
   "triggers-database":"%%mlFinalTriggersDbName%%",
   "triple-index":true,
   "collection-lexicon":true,
   "uri-lexicon":true,
   "range-field-index":[
      {
         "scalar-type":"dateTime",
         "field-name":"systemStart",
         "collation":"",
         "range-value-positions":true,
         "invalid-values":"reject"
      },
      {
         "scalar-type":"dateTime",
         "field-name":"systemEnd",
         "collation":"",
         "range-value-positions":true,
         "invalid-values":"reject"
      }
   ],
   "field":[
      {
         "field-name":"systemStart",
         "metadata":"true",
         "word-lexicons":null,
         "included-elements":null,
         "excluded-elements":null,
         "tokenizer-overrides":null
      },
      {
         "field-name":"systemEnd",
         "metadata":"true",
         "word-lexicons":null,
         "included-elements":null,
         "excluded-elements":null,
         "tokenizer-overrides":null
      }
   ]
}
```

For this configuration, the "range-field-index" and "field" configuration elements are not deployed.

output attached

#### Tech details

Which Operating System are you using? Mac

Which version of MarkLogic are you using? 9.0.9-1

Which version of the Data Hub Framework are you using? 5.0.0

**Comment by derms** [comment]
output of command -

/Users/dsmyth/dev/workspace/PDH_OFF_411/myoutput.txt

**Comment by derms** [comment]
adding output 

[myoutput.txt](https://github.com/marklogic/marklogic-data-hub/files/3206707/myoutput.txt)

**Comment by derms** [comment]
fields that are actually deployed - the systemStart and systemEnd fields are not deployed
![image](https://user-images.githubusercontent.com/1885934/58164344-f25a2780-7c85-11e9-8dec-3b859258181a.png)

**Comment by derms** [comment]
@rjrudin ok great thanks for letting me know

**Comment by aebadirad** [comment]
Fixed in 5.0.1
